requirements-tests.txt

```diff
@@ -15,3 +15,4 @@ termcolor>=2
 tomli==2.0.1
 tomlkit==0.11.6
 types-pyyaml
+types-six
```

Comment:
You've got the right idea, but you need to add this dependency to a `requires` field in the `METADATA.toml` file, rather than adding it globally :)

E.g. https://github.com/python/typeshed/blob/5c3f10e1df3d6ad86d9d725aa42d0512d7b86ad9/stubs/Flask-SQLAlchemy/METADATA.toml#L2

Reply:
Thanks. Done.
Can't you hint what to do if I need types from module having `py.typed`?
In particular I'm speaking about #9304 and `click` module.